package.json

```diff
@@ -107,6 +107,7 @@
     "yargs": "^1.3.1"
   },
   "dependencies": {
+    "@fingerprintjs/fingerprintjs": "^3.3.0",
```

Comment:
No, It's just the package scope name, It is published under @fingerprintjs organization on npm

Reply:
Does this make sense to include as a package dependency for base Prebid, given this will be a large portion of code that will only be used if users include your id module in their build. Normally id modules provide an endpoint that returns an Id to be set to local storage when the getId function is called. Would you be able to support this?